packages/babel-plugin-transform-regenerator/test/arrow-inside-generator.js

```diff
@@ -0,0 +1,19 @@
+const assert = require("assert");
+const babel = require("babel-core");
+
+test("Doesn't throw with arrow function inside generator", function () {
```

Comment:
Focusing on 7.x seems best at this point. If you just want to test the input/output without `exec.js` you can go the `expect.js` and `actual.js` route too, not strong feelings really.